SygicUIKit/Classes/Views/Search/SYUISearchBarController.swift

```diff
@@ -42,6 +60,9 @@ public class SYUISearchBarController: UIViewController, SYUISearchBarDelegate {
         searchBarView.searchBarDelegate = self
     }
     
+    /// Prefill search input field.
+    ///
+    /// - Parameter text: Text for search input field.
     public func prefillSearch(with text: String) {
         guard let searchView = searchBarView as? SYUISearchBarView else { return }
```

Comment:
cize ked si tutok injectnes vlstny searchBarView tak ti neprefilne text? asi by sme to mali zohladnit v SearchBarProtocole